src/build/base64-impl-browser.ts

```diff
@@ -0,0 +1,10 @@
+
+export function base64Encode (str: string): string {
+  return btoa(String.fromCharCode(...new TextEncoder().encode(str)))
```

Comment:
Try avoid TextEncoder and Uint8Array as we don’t heavily depend on them. Use escape and decodeURIComponent trick for better compatibility.